README.md

```diff
@@ -77,27 +77,29 @@ yarn download:plugins
 yarn tracecompass-server:download
 ```
 
-For example, for a working dev environment there is no need to download the plugins and the tracecompass-server.
+Note: Downloading plugins and tracecompass-server is optional but recommended for full functionality.
 
-When working with the browser variant the [Docker based development](#docker-based-development) can be used, which automates this step.
+In the Docker-based development environment, this step is automated.
```

Comment:
Any specific reason to not have the internal link here?

Reply:
No feel free to re-add this.